geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geohash/GeohashUtils.scala

```diff
@@ -59,12 +59,12 @@ object GeohashUtils
    */
   case class ResolutionRange(minBitsResolution:Int=5,
                              maxBitsResolution:Int=63,
-                             numBitsIncrement:Int=2)
-    extends Inclusive(minBitsResolution, maxBitsResolution, numBitsIncrement) {
+                             numBitsIncrement:Int=2) {
```

Comment:
Hmm... any chance this represents an API change we care about?

Reply:
i would say no... `Inclusive` is final in scala 2.12 so we can't extend it any more, hence the change.